code/game/gamemodes/technomancer/spells/mend_organs.dm

```diff
@@ -35,7 +35,7 @@
 				if(O.damage > 0) // Fix internal damage
 					O.heal_damage_i(heal_power / 2)
 				if(O.damage <= 5 && O.organ_tag == O_EYES) // Fix eyes
-					H.sdisabilities &= ~SDISABILITY_NERVOUS
+					H.remove_blindness_source( "broken eyes")
```

Comment:
it's a good idea to #define al ltrait sources

use the DATUM_TRAIT pattern and put it in the mob.dm file ofc